test/e2e/podtatohead_test.go

```diff
@@ -0,0 +1,214 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/stretchr/testify/require"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestPodtatoheadEvaluation(t *testing.T) {
+	if !isE2ETestingAllowed() {
+		t.Skip("Skipping TestHelloWorldDeployment, not allowed by environment")
+	}
+
+	// Setup the E2E test environment
+	testEnv, err := newTestEnvironment(
+		"../events/podtatohead.deploy-v0.1.1.triggered.json",
+		"../shipyard/podtatohead.deployment.yaml",
+		"../data/podtatohead.jes-config.yaml",
+	)
+
+	require.NoError(t, err)
+
+	additionalResources := []struct {
+		FilePath     string
+		ResourceName string
+	}{
+		{FilePath: "../data/podtatoserver-0.1.0.tgz", ResourceName: fmt.Sprintf("charts/%s.tgz", testEnv.EventData.Service)},
+		{FilePath: "../data/locust.basic.py", ResourceName: "locust/basic.py"},
+		{FilePath: "../data/locust.conf", ResourceName: "locust/locust.conf"},
+		{FilePath: "../data/podtatohead.sli.yaml", ResourceName: "prometheus/sli.yaml"},
+		{FilePath: "../data/podtatohead.slo.yaml", ResourceName: "slo.yaml"},
+		{FilePath: "../data/podtatohead.remediation.yaml", ResourceName: "remediation.yaml"},
+	}
+
+	err = testEnv.SetupTestEnvironment()
+	//	require.NoError(t, err)
```

Comment:
Are these lines still relevant?

Reply:
Hmm this seems to be an oversight from my part, that code shouldn't be commented out ...